README.md

```diff
@@ -85,6 +85,10 @@ Name | Type | Required | Description
 -----|------|----------|-------------
 `table`   | string | yes | table to write points into
 
+## Numbers
+
+When reading numircal data types of variable storage size (see http://www.postgresql.org/docs/9.1/static/datatype-numeric.html), values are returned as type `String`. You may want to use http://juttle.github.io/juttle/modules/number/#numberfromstring function in your `juttle` if your data has numbers of these types.
```

Comment:
s/numircal/numerical

And let's add:

Fixed precision types such as `double precision` and `real` are returned as number type in Juttle.